.gitignore

```diff
@@ -31,3 +31,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+### Docker ###
+.env
```

Comment:
![image](https://github.com/user-attachments/assets/d96f9380-690f-47c5-8abd-b89c4369aef3)
you use this file in docker but in repository you ignore it

Reply:
So I should delete this line?